packages/rps/package.json

```diff
@@ -20,7 +20,7 @@
     "postinstall": "rm -f node_modules/web3/index.d.ts",
     "prettier:check": "npx prettier --check 'src/**/*.{ts,tsx}'",
     "prettier:write": "npx prettier --write 'src/**/*.{ts,tsx}'",
-    "start": "npm run clearContracts && node scripts/start.js",
+    "start": "yarn contracts:compile && node scripts/start.js",
```

Comment:
Not sure that we ever really need `clearContracts` ? We could remove that script if it is unnecessary.

Reply:
I think it would be needed if we ever deleted a contract?